lib/ae_mdw_web/controllers/tx_controller.ex

```diff
@@ -112,14 +112,23 @@ defmodule AeMdwWeb.TxController do
     end
   end
 
-  defp read_tx_hash(tx_hash) do
+  defp read_tx_hash(tx_hash, wrapper_lookup? \\ false) do
```

Comment:
From what it looks like this function is always going to be called with a single parameter, so the 2nd parameter will always be `false`. Perhaps we can get rid of it?

Reply:
It's used to search for inner transaction. 
The sequence is to first try like an outer, most common case (by far). If doesn't find, looks up like an inner transaction.
We could create also a new endpoint for inner transactions or add an additional parameter but avoided because it will be less transparent with one more decision for the client (possibly implying in an additional request).

Reply:
Not applicable anymore as the /tx/:hash is not part of the scope.